pennylane_lightning/lightning_gpu/_adjoint_jacobian.py

```diff
@@ -71,13 +92,81 @@ def _adjoint_jacobian_dtype(self):
 
         Returns: the AdjointJacobian class
         """
-        jacobian_lightning = (
-            AdjointJacobianC64() if self.dtype == np.complex64 else AdjointJacobianC128()
-        )
-        create_ops_list_lightning = (
-            create_ops_listC64 if self.dtype == np.complex64 else create_ops_listC128
-        )
-        return jacobian_lightning, create_ops_list_lightning
+        if self._use_mpi:
+            jacobian_lightning = (
+                AdjointJacobianMPIC64() if self.dtype == np.complex64 else AdjointJacobianMPIC128()
+            )
+            create_ops_list_lightning = (
+                create_ops_listMPIC64 if self.dtype == np.complex64 else create_ops_listMPIC128
+            )
+            return jacobian_lightning, create_ops_list_lightning
+        else:  # without MPI
+            jacobian_lightning = (
+                AdjointJacobianC64() if self.dtype == np.complex64 else AdjointJacobianC128()
+            )
+            create_ops_list_lightning = (
+                create_ops_listC64 if self.dtype == np.complex64 else create_ops_listC128
+            )
+            return jacobian_lightning, create_ops_list_lightning
+
+    def _process_jacobian_tape(self, tape: QuantumTape, split_obs: bool = False, use_mpi=False):
+        """Process a tape, serializing and building a dictionary proper for
+        the adjoint Jacobian calculation in the C++ layer.
+
+        Args:
+            tape (QuantumTape): Operations and measurements that represent instructions for execution on Lightning.
+            split_obs (bool, optional): If splitting the observables in a list. Defaults to False.
```

Comment:
split_obs (bool, optional): If splitting the observables in a list. Defaults to False.

```suggestion
            split_obs (bool, optional): If splitting the observables in a list. Defaults to False.
            use_mpi (bool, optional): If distributing computation with MPI. Defaults to False.
```